<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Universidad</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>301</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Datos del Estudiante:</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nombres:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Apellidos:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Codigo:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbciclo">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Ciclos</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>I</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>II</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>III</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>IV</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>V</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>VI</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Otros</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ciclo:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tcodigo">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="tnombre">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="tapellido">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="bcurso">
     <property name="geometry">
      <rect>
       <x>194</x>
       <y>160</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ver Cursos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="blimpiar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Limpiar</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>471</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Cursos</string>
    </property>
    <widget class="QLabel" name="lblcurso">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>441</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>771</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Notas de Cursos:</string>
    </property>
    <widget class="QLabel" name="lblc1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Curso 1:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblc2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Curso 2:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblc3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Curso 3:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblc4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Curso 4:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="tnota1">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="tnota2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>70</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="tnota3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>110</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="tnota4">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>150</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>140</y>
       <width>321</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Observacion:</string>
     </property>
     <widget class="QLabel" name="lblobservacion">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>281</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>210</y>
       <width>211</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Promedio</string>
     </property>
     <widget class="QLabel" name="lblpromedio">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="bcalcular">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>50</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>CALCULAR </string>
     </property>
    </widget>
    <widget class="QPushButton" name="blimpiartodo">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>50</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>LIMPIAR TODO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bcalcular_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>180</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>LIMPIAR NOTAS</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="bnotas">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>210</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Digitar Notas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bsalir">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SALIR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>RECORD DE NOTAS</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
